File: ts/iq/privacy~get~privacy.ts
/** <iq type="get" xmlns="privacy">. Replies with one of: SuccessLID, Success, Error. */
export function buildPrivacyGetContactBlacklist(): BinaryNode {
  return { tag: 'iq', attrs: { to: 's.whatsapp.net', xmlns: 'privacy', type: 'get' }, content: [
      { tag: 'privacy', attrs: {  } },
    ] }
}

File: rust/iq/privacy~get~privacy.rs
/// <iq type="get" xmlns="privacy">. Replies with one of: SuccessLID, Success, Error.
pub fn build_privacy_get_contact_blacklist() -> Node {
    NodeBuilder::new("iq")
        .attr("to", "s.whatsapp.net")
        .attr("xmlns", "privacy")
        .attr("type", "get")
        .child(NodeBuilder::new("privacy").build())
        .build()
}